packages/babel-preset-mc-app/index.js

```diff
@@ -127,10 +127,7 @@ module.exports = function getBabePresetConfigForMcApp() {
         },
       ],
       require('@babel/plugin-proposal-do-expressions').default,
-      require('@babel/plugin-proposal-optional-chaining').default,
-      require('@babel/plugin-proposal-nullish-coalescing-operator').default,
       // Adds syntax support for import()
-      require('@babel/plugin-syntax-dynamic-import').default,
```

Comment:
Ah you removed this but remove the dependency of `babel-plugin-transform-dynamic-import`.

Which one should we remove?

Reply:
Good point. The changelog is not the most expressive on this. 

<img width="539" alt="CleanShot 2020-01-20 at 09 38 58@2x" src="https://user-images.githubusercontent.com/1877073/72711118-b787f680-3b68-11ea-910b-5bb0bc90c19f.png">

Somehow it sounds to me as the support it natively now. I removed both. Will build and give it a spin locally and see what happens.

Reply:
So the tests locally with this version of the preset pass. Also a production build can be made. Anything else besides the later canary build we should test before moving on?

Reply:
If everything passes, it should be fine